### DISCUSSION TITLE
Sample front-end site for GitHub App?

### DISCUSSION THREAD

**Discussion by drok** [community/community#36212] [Other Feature Feedback, Questions, & Ideas]
Hello,

I am developing a GitHub App; the back-end is nearly complete, but I'm a bit lost in finding samples of front-end websites; probably because the generic search terms.

What are some good ways to search for "Hello World" type of web front ends that demonstrate [Customer Experience Best Practices for GitHub Apps](https://docs.github.com/en/developers/github-marketplace/creating-apps-for-github-marketplace/customer-experience-best-practices-for-apps) ?

**Comment by ossanna16** [comment]
Hey there @drok and thank you for asking a good question! I'm glad you found our community 🙂
